src/java.base/share/classes/java/lang/Math.java

```diff
@@ -1451,6 +1451,7 @@ public static int max(int a, int b) {
      * @param   b   another argument.
      * @return  the larger of {@code a} and {@code b}.
      */
+    @HotSpotIntrinsicCandidate
```

Comment:
Is this required or was made in sake of consistency?

Reply:
This annotation is checked in src/hotspot/share/classfile/classFileParser.cpp:5358 so it is required in case the CheckIntrinsics parameter is set to true (which is the default value, I suppose)

Reply:
Thanks, agree, it's good to know